Neos.Neos/Documentation/CreatingASite/ContentDimensions.rst

```diff
@@ -224,7 +225,7 @@ The default preset can have an empty `uriSegment` value. The following example w
             'en':
               label: 'English (US)'
               values: ['en_US']
-              uriSegment: ''
+              uriSegment: 'en'
```

Comment:
This is not correct (AFAIK), you still need to have the uriSegment empty, in addition to `supportEmptySegmentForDimensions` (which is the default)

Reply:
Yeah supportEmptySegmentForDimensions is true by default. In my customer instance and test instance I did not make it with the way the documentation tells to handle that and therefore I talked with some guys on slack and in the end I configure it like this and now I am  able to surf the page with and without the default uriSegment and I get the content but if I have an empty segment then I end up in 404.

But if this is wrong then I am looking forward to find the right way and try it.